AWSComprehendTests/AWSComprehendTests.swift

```diff
@@ -138,7 +138,7 @@ class AWSComprehendTests: XCTestCase {
         let comprehendClient = AWSComprehend.default()
         let detectSentimentRequest = AWSComprehendDetectSentimentRequest()
         detectSentimentRequest!.languageCode = AWSComprehendLanguageCode.en
-        detectSentimentRequest!.text = "I have no strong feelings one way or the other"
```

Comment:
This sentence is no longer neutral enough for AWS Comprehend and it's now detecting it as mostly negative.